django_cloud_deploy/workflow/__init__.py

```diff
@@ -323,3 +328,14 @@ def _generate_base_secrets(database_username: str,
             secrets: Base secret data used by kubernetes.
         """
         return {'username': database_username, 'password': database_password}
+
+    @staticmethod
+    def _load_secret_names(
+            required_service_accounts:
+            Optional[Dict[str, List[Dict[str, Any]]]] = None) \
+            -> (List[str], List[str]):
```

Comment:
Does something like the following work:

def _load_secret_names(
            required_service_accounts:
            Optional[Dict[str, List[Dict[str, Any]]]] = None
)  -> (List[str], List[str]):

You make a change similar with above to function _generate_secrets. They should be consistent.